speedometer.py

```diff
@@ -495,7 +502,7 @@ def readable_speed(speed):
     if speed == None or speed < 0: speed = 0
 
     units = "B/s  ", "KiB/s", "MiB/s", "GiB/s", "TiB/s"
-    step = 1L
+    step = long(1)
```

Comment:
I don't think there was a reason to use 1L here (unless we're relying on the repr), how about using normal ints throughout and let python2 upgrade them as required